core/src/main/java/org/elasticsearch/index/mapper/geo/BaseGeoPointFieldMapper.java

```
@@ -344,6 +347,14 @@ public void setLatLonEnabled(MappedFieldType latFieldType, MappedFieldType lonFi
         public IndexFieldData.Builder fielddataBuilder() {
             return new AbstractGeoPointDVIndexFieldData.Builder();
         }
+
+        @Override
+        public ValueFormat valueFormat(@Nullable String format, DateTimeZone timeZone) {
+            if (format != null) {
+                throw new IllegalArgumentException("Field [" + name() + "] of type [" + typeName() + "] does not support custom formats");
+            }
```

should we also throw an exception if timeZone is specified here?